### ISSUE TITLE
undefined symbol: napi_add_finalizer

### ISSUE THREAD

**Issue by averri** [lovell/sharp#2117]
I have upgraded the library from `0.24.1` to `0.25.0` because the problems with the former. I have removed `node_modules` and run `yarn install`. When running the application this is what I get:

```
/usr/bin/node: symbol lookup error: /home/alex/dev/src/other/strenux/strenux-api/node_modules/sharp/build/Release/sharp.node: undefined symbol: napi_add_finalizer
```

I cannot do anything, Sharp has broken my project.

Node: 12.16.0
OS: Ubuntu 18

**Comment by lovell** [comment]
Hi, it looks like some of the prebuilt binaries for v0.25.0 erroneously used N-API v5 rather than v4. I'll roll out a fix for this in the next hour or so.

**Comment by lovell** [comment]
Although there is a problem, I don't think it should affect you as N-API v5 was added in Node.js 12.11.0 and you say you're using Node.js 12.16.0.

Are you sure about the version being used at runtime, as I would not expect this error from Node.js 12.16.0? I notice you've commented on other issues recently about runtime problems - perhaps you have multiple, conflicting versions of Node.js made available at runtime?

**Comment by lovell** [comment]
v0.25.1 pins N-API to v3 and is now available, thanks for reporting.

**Comment by averri** [comment]
Hi @lovell, thanks for informing. I have tested the new version v0.25.1 with Node 12.16.0 and 13.8.0 and it's working fine after a clean install (removed node_modules and did `yarn install`).

I was using Node 12.16.0 and upgraded recently to 13.8.0.

I do use nvm indeed for managing different Node.js versions. I never had problems with nvm and Sharp before, I started seeing these issues recently (few weeks ago).
